core-data/models.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+const models = [
+	{ name: 'postType', kind: 'root', pk: 'slug', baseUrl: '/wp/v2/types' },
```

Comment:
It should be `type` rather than `kind` for consistency. I guess this is because of the fact that `type` is a reserved word in actions, right?

Reply:
Yes it's one of the reasons. Also, type is overused :) so we could have a postype of type type :)

Reply:
Can we have a post type of kind type? :D